packages/create-turbo/src/transforms/index.ts

```diff
@@ -0,0 +1,13 @@
+import { transform as packageManagerTransform } from "./package-manager";
+import { transform as internalTransform } from "./internal-example";
+import { transform as gitIgnoreTransform } from "./git-ignore";
+import type { TransformInput, TransformResult } from "./types";
+
+/**
+ * In the future, we may want to support sourcing additional transforms from the templates themselves.
+ */
+export const transforms: Array<(args: TransformInput) => TransformResult> = [
+  internalTransform,
```

Comment:
How about `coreExample`? "internal is a bit unclear to me" since it's not internal, we're intending external usage

Reply:
internal means internal to the repo. It's an example that comes from `vercel/turbo`. I like internal a bit more than core because it carries with it some information about the example itself (I don't know what a core example is just by looking at the name), but I can try and come up with something more formal.